pkgs/development/python-modules/python-nvd3/default.nix

```diff
@@ -1,20 +1,18 @@
-{ lib, buildPythonPackage, fetchPypi, python-slugify, jinja2, setuptools }:
+{ lib, buildPythonPackage, fetchFromGitHub, python-slugify, jinja2, setuptools, pytest }:
 
 buildPythonPackage rec {
   pname = "python-nvd3";
   version = "0.15.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "05bph75x6ss0nbavwylsbam28h4bvh88nfjgmas5bwhfgvs5zmzv";
+  src = fetchFromGitHub {
+    owner = "areski";
+    repo = "python-nvd3";
+    rev = "dc8e772597ed72f413b229856fc9a3318e57fcfc";
+    sha256 = "1vjnicszcc9j0rgb58104fk9sry5xad1xli64jana9bkx42c6x1v";
   };
 
   propagatedBuildInputs = [ python-slugify jinja2 setuptools ];
-
-  # The package in PyPI includes no testing support; only the development git version can be tested.
-  doCheck = false;
-
-  pythonImportsCheck = [ "nvd3" ];
+  checkInputs = [ pytest ];
```

Comment:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

Reply:
This doesn't work - no tests are run. Judging by their .travis.yml, the correct command for tests is `coverage run --source=nvd3 setup.py test`, so I guess I'll just put that into `checkPhase`.